clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -154,6 +155,38 @@ public class AzureComputeClient extends AzureBaseClient {
     null
   }
 
+  ServiceResponse<Void> destroyServerGroup(String resourceGroupName, String serverGroupName) {
+    ServiceResponse<Void> result = null
+    // The API call might return a timeout exception or some other Azure CloudException that is not
+    //   related to the operation we are trying to execute; retry couple of times and if the final
+    //   retry fails then rethrow
+    long operationRetry = 0
+    while (operationRetry < AZURE_ATOMICOPERATION_RETRY) {
```

Comment:
Makes sense; I can add the sleeps in between the calls.
BTW can you point me into AmazonClientInvocationHandler what method should I check? I did not notice anyone that has similar retries.